src/js/components/Person/AddPersonDrawerMainContent.jsx

```diff
@@ -86,21 +95,30 @@ const AddPersonDrawerMainContent = () => {
       teamName,
     };
     mutate(makeRequestParams(plainParams, {}));
-    // This removes the recently "added" staff, from the list of staff who can be added, the staffToDisplayList
-    const updatedStaffToDisplayList = staffToDisplayList.filter((staff) => staff.id !== person.id);
-    setStaffToDisplayList(updatedStaffToDisplayList);
+    // Remove this staff from the All Staff less Adds list (since they were added to the team)
+    const updatedAllStaffLessAdds = allStaffLessAdds.filter((staff) => staff.id !== person.id);
+    setAllStaffLessAdds(updatedAllStaffLessAdds);
+    if (staffToDisplayList && staffToDisplayList.length > 0) {
+      // If we are currently showing filtered, remove this staff member from the filtered Staff To Display List
+      const updatedStaffToDisplayList = staffToDisplayList.filter((staff) => staff.id !== person.id);
+      setStaffToDisplayList(updatedStaffToDisplayList);
+    }
   };
 
   return (
     <AddPersonDrawerMainContentWrapper>
       <SearchBarWrapper>
-        <SearchBar2024
+        <TextField
```

Comment:
When you get a moment, I would love to maintain the use of a component (perhaps `SearchBar2025`?) for the actual search bar input field (instead of an inline TextField), because we want features like an "x" to delete the search text, a search icon in the box, etc.